solutions/kubernetes/cortx-deploy-functions.sh

```diff
@@ -43,6 +45,21 @@ function download_deploy_script(){
     popd
 }
 
+#On master
+#download CORTX support bundle log scripts
+
+function download_log_script(){
+    rm -rf /var/tmp/parse_scripts
+    rm -rf /var/tmp/logs-cortx-cloud.sh
+
+    mkdir -p /var/tmp/parse_scripts
+    wget $LOG_SCRIPT -P /var/tmp
```

Comment:
We should use logs-cortx-cloud.sh from the same version used for deployment. https://github.com/Seagate/cortx-k8s/blob/stable/k8_cortx_cloud/logs-cortx-cloud.sh

Reply:
Done